### PR TITLE
chore: Upgrade istio, kiali, jaeger addons

### PR DESCRIPTION
**Pull request by GoelDeepak**
**What type of PR is this?**
Chore

**What this PR does/ why we need it**:
This PR has three commits each for kiali, jaeger and istio. Istio 1.7+ no longer ship kiali and jaeger, therefore, this PR create an independent addons for kiali and jaeger. 

**Which issue(s) this PR fixes**:
https://jira.d2iq.com/browse/D2IQ-73677

**Special notes for your reviewer**:
istio chart upgrade PR https://github.com/mesosphere/charts/pull/929

**Does this PR introduce a user-facing change?**:
yes

```release-note
Istio has been upgraded from 1.6.12 to 1.8.1 
* Kiali and Jaeger are now independent addons and needs to be enabled explicitly
```

**Checklist**

* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: addons/istio/istio.yaml [hunk 1]
```diff
@@ -6,17 +6,9 @@ metadata:
...
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.6.12-1"
-    appversion.kubeaddons.mesosphere.io/istio: "1.6.12"
-    appversion.kubeaddons.mesosphere.io/kiali: "1.18.0"
-    appversion.kubeaddons.mesosphere.io/jaeger: "1.16.0"
-    stage.kubeaddons.mesosphere.io/kiali: Preview
-    stage.kubeaddons.mesosphere.io/jaeger: Preview
-    endpoint.kubeaddons.mesosphere.io/kiali: "/ops/portal/kiali"
-    endpoint.kubeaddons.mesosphere.io/jaeger: "/ops/portal/jaeger"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.8.1-1"
```
Comment (vespian) [inline addons/istio/istio.yaml:9]: Do we want to wait for https://github.com/mesosphere/charts/pull/943 to land?

File: addons/jaeger/jaeger.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - name: vsphere
+      enabled: false
+  chartReference:
+    chart: jaeger-operator
+    repo: https://jaegertracing.github.io/helm-charts/
+    version: 2.18.3
+    values: |
+      image:
+        repository: jaegertracing/jaeger-operator
+        tag: 1.20.0
```
Comment (vespian) [inline addons/jaeger/jaeger.yaml:40]: why not 1.21.0?

https://github.com/jaegertracing/jaeger/releases

**Review by vespian — APPROVED** [review]
Left few comments.

File: addons/istio/istio.yaml [hunk 3]
```diff
@@ -42,14 +34,9 @@ spec:
   chartReference:
     chart: istio
     repo: https://mesosphere.github.io/charts/staging
-    version: 1.6.12
+    version: 1.8.1
```
Comment (gracedo) [inline addons/istio/istio.yaml:37]:
```suggestion
    version: 1.8.2
```
@GoelDeepak should we bump this to latest version?

File: addons/istio/istio.yaml [hunk 4]
```diff
@@ -6,17 +6,9 @@ metadata:
...
-    stage.kubeaddons.mesosphere.io/kiali: Preview
-    stage.kubeaddons.mesosphere.io/jaeger: Preview
-    endpoint.kubeaddons.mesosphere.io/kiali: "/ops/portal/kiali"
-    endpoint.kubeaddons.mesosphere.io/jaeger: "/ops/portal/jaeger"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.8.1-1"
+    appversion.kubeaddons.mesosphere.io/istio: "1.8.1"
     docs.kubeaddons.mesosphere.io/istio: "https://istio.io/docs/"
-    docs.kubeaddons.mesosphere.io/kiali: "https://istio.io/docs/tasks/telemetry/kiali/"
-    docs.kubeaddons.mesosphere.io/jaeger: "https://istio.io/docs/tasks/telemetry/distributed-tracing/jaeger/"
     values.chart.helm.kubeaddons.mesosphere.io/istio: "https://raw.githubusercontent.com/mesosphere/charts/a7d3c02bac8c4c3ae02854e60483b97cfb80b1b7/staging/istio/values.yaml"
```
Comment (gracedo) [inline addons/istio/istio.yaml:12]:
```suggestion
    values.chart.helm.kubeaddons.mesosphere.io/istio: "https://raw.githubusercontent.com/mesosphere/charts/1079b3c/staging/istio/values.yaml"
```
just a values change grabbed from #775

**Review by joejulian — APPROVED** [review]

**Pull request merged**